scalyr_agent/monitor_utils/k8s.py

```diff
@@ -1218,8 +1217,7 @@ def query_api( self, path, pretty=0 ):
 
         url = self._http_host + path + pretty
 
-        import traceback
-        global_log.info('k8s api get %s, %s' % (url, traceback.format_exc()))
+        global_log.info('query_api: %s \n %s' % (url, traceback.format_exc()))
```

Comment:
Should we be using `format_exc()` here?  We are not in an exception - maybe `format_stack()`?